docs/ivys.js

```diff
@@ -90,8 +90,8 @@ var ProductCollection = function () {
     self.all_products = {}; // sku number : 'product name'
     self.favorite_products = {}; // sku number : 'product name'
     self.all_tags = {}; // 'tag-id' : 'tag-name'
-    self.selected_tags = {}; // 'tag-name' : true or false ?
-    self.selected_products = {}; // 'product name' : true or false ?
+    self.selected_tags = {}; // 'tag-name' : true or false
+    self.selected_products = {}; // 'product SKU number' : true or false
```

Comment:
I think true or false works for selected_tags and selected_products. What do you think? I also figured it's better to use the product's SKU number